setup.py

```diff
@@ -27,6 +28,12 @@
     'pytest',
 ]
 
+# WARNING: This imposes limitations on test/requirements.txt such that the
+# full Pip syntax is not supported. See also
+# <http://stackoverflow.com/questions/14399534/>.
+with open(os.path.join('test', 'requirements.txt')) as f:
```

Comment:
Nit: `with open('test/requirements.txt') as f:`. Works cross platform with Window paths if that's what you were worried about.